### DISCUSSION TITLE
Is there a keyboard shortcut for "Edit this note" ?

### DISCUSSION THREAD

**Discussion by Potjoe-97** [TriliumNext/Trilium#2171] [Q&A] [answered]
Hi!

I use Trilium to follow my lessons, and despite the fact that I split them often, they are still too long for not become read-only by default. I don't mind to click the Edit this note button, this is a behaviour that I like (keep me from editing the note when I just want to read through it), but is there a keyboard shortcut to trigger the edit mode ?

Thank you for your help.

**Comment by zadam** [comment, marked as answer]
Not by default but you can assign one:

![image](https://user-images.githubusercontent.com/617641/133406796-99579cc6-e587-41c9-adfd-0eab1b9c29d1.png)

**Reply by Potjoe-97** [reply]
Thank you ! It's weird, the action is not listed in keyboard shortcuts tab. Like you, I typed readonl in the search field, and there is no result... I'm on 0.47.7

**Reply by zadam** [reply]
I see, I didn't realize this was implemented in the master branch (unreleased 0.48) and is not available in the 0.47.

**Reply by Potjoe-97** [reply]
I'll wait for next releases now that I know it's there. Thanks!
